### DISCUSSION TITLE
"start service after after v3.0.0"

### DISCUSSION THREAD

**Discussion by doublex** [vesoft-inc/nebula#4826] [Q&A] [answered]
`nebula.service start all` tells (but everything works):

```` 
[WARN] nebula-storaged after v3.0.0 will not start service until it is added to cluster.
[WARN] See Manage Storage hosts:ADD HOSTS in https://docs.nebula-graph.io/
````

How to get rid of this message?

**Comment by QingZ11** [comment, marked as answer]
As the prompt says, you need to run ` add  hosts` command, just like [https://docs.nebula-graph.io/3.3.0/4.deployment-and-installation/manage-storage-host/](https://docs.nebula-graph.io/3.3.0/4.deployment-and-installation/manage-storage-host/)
